rest-api/test/unit_test/api_test/questionnaire_response_api_test.py

```diff
@@ -225,6 +224,99 @@ def test_demographic_questionnaire_responses(self):
               }
     self.assertJsonResponseMatches(expected, summary)
 
+  def test_consent_with_extension_language(self):
+    with FakeClock(TIME_1):
+      participant_id = self.create_participant()
+      self.send_consent(participant_id, consent_language='es')
```

Comment:
We should also have a test that verifies when you submit a questionnaire response that is *not* consent, that the language doesn't change.